src/main/kotlin/org/taymyr/lagom/elasticsearch/search/dsl/query/term/DateRangeExpression.kt

```diff
@@ -0,0 +1,52 @@
+package org.taymyr.lagom.elasticsearch.search.dsl.query.term
+
+import java.time.temporal.ChronoUnit
+
+class DateRangeExpression private constructor() {
+    private var anchor: String = ""
+    private var expression: String? = ""
```

Comment:
Why `expression` is a nullable type with default initialization?

Reply:
fixed to non-nullable